controllers/reviewController.js

```diff
@@ -1,43 +1,73 @@
 const { reviewService } = require("../services");
 const { asyncWrap } = require("../middleware/errorControl")
 
+const getReview = asyncWrap(async (req, res) => {
+    const { productId } = req.params;
+    const { userId } = req.body
+    if ( !productId ) {
+        const err = new Error("KEY_ERROR");
+        err.statusCode = 400;
+        throw err;
+    }
+    const review = await reviewService.getReview( productId, userId );
+    return res.status(200).json({review: review})
```

Comment:
코드 사이에 간격을 추가해서 가독성을 높여주세요-!

```suggestion
    const { productId } = req.params;
    const { userId } = req.body
    
    if ( !productId ) {
        const err = new Error("KEY_ERROR");
        err.statusCode = 400;
        throw err;
    }
    
    const review = await reviewService.getReview( productId, userId );
    return res.status(200).json({review: review})
```